client/core/core_test.go

```diff
@@ -307,7 +308,7 @@ func (tdb *TDB) CreateAccount(ai *db.AccountInfo) error {
 
 func (tdb *TDB) DisableAccount(ai *db.AccountInfo) error {
 	tdb.accts = nil
```

Comment:
From what I can tell, the `TDB.accts` field is completely unused. What would be better here is something like

```
tdb.disabledAcct = ai
```

and then you can check that the request got through in the tests.